lib/simulator_avx.h

```diff
@@ -153,6 +153,7 @@ class SimulatorAVX final {
       break;
     default:
       // Cannot reach here.
+      ml = _mm256_set_epi32(0, 0, 0, 0, 0, 0, 0, 0);
```

Comment:
It's probably preferable to raise an error here, if we have a way to do so.

(Similarly for other "unreachable" sections in this file)

Reply:
How do you propose to raise an error here? These sections are really unreachable if those functions are used correctly. The correct usage is described in function comments.

Reply:
Sorry, I was in a Python state of mind. This should be fine as-is.